packages/api-fetch/src/index.js

```diff
@@ -31,10 +31,11 @@ function checkCloudflareError( error ) {
 function apiFetch( options ) {
 	const raw = ( nextOptions ) => {
 		const { url, path, body, data, parse = true, ...remainingOptions } = nextOptions;
-		const headers = remainingOptions.headers || {};
-		if ( ! headers[ 'Content-Type' ] && data ) {
-			headers[ 'Content-Type' ] = 'application/json';
-		}
+		const headers = {
+			Accept: 'application/json, */*;q=0.1',
+			'Content-Type': 'application/json',
```

Comment:
Image uploads are not JSON, this could have broke image uploads in media blocks.

Reply:
@youknowriad Sorry, not sure I follow. What's your point?

Is it:

* That we should have e2e tests covering image uploads?
* That we should have unit tests in this package ensuring multiple content types are supported?
* Something else?

Reply:
Just that uploading images is broken in master now because of this :)

Reply:
I'm working on a fix currently.

Reply:
Fix at #11437